front/index.html

```diff
@@ -5,17 +5,33 @@
     <link rel="icon" href="/favicon.ico" />
```

Comment:
I think this file is not used actually. Can you make all your changes in front/public/index.html and remove this front/index.html file?

Reply:
Yeah - haha was wondering why there were two index.htmls. Will do!